tests/main/high-user-handling/task.yaml

```diff
@@ -1,20 +1,24 @@
 summary: Check that the refresh data copy works.
 
-systems: [-ubuntu-core-*]
+systems:
+    - -ubuntu-14.04-*  # no support for tests.session
+    - -ubuntu-core-*  # no support for useradd
```

Comment:
UC does have useradd though ?

Reply:
Yeah, but that's an existing limitation. I just documented it.